server/src/parser/table-reader.ts

```diff
@@ -57,7 +57,7 @@ class LineReader {
 
   public readLine(lineNumber: number, line: string) {
     this.lineNumber = lineNumber;
-    this.line = this.trimComments(line);
+    this.line = this.trimPipes(this.trimComments(line));
```

Comment:
Couldn't you just remove trimPipes entirely since endOfCellIdx's cellSeparators variable would theoretically handle the highlighting? Not sure if this is truly the case, just a Robot enthusiast hoping to eat up this feature ASAP :)

Reply:
Without handling the first pipe the parser would think that every first cell starts with a `|`. Therefore we must handle it and also care about an empty first cell